pkg/kubelet/dockertools/manager.go

```diff
@@ -1877,7 +2075,7 @@ func (dm *DockerManager) SyncPod(pod *api.Pod, _ api.PodStatus, podStatus *kubec
 				continue
 			}
 		}
-		glog.V(4).Infof("Creating container %+v in pod %v", container, format.Pod(pod))
+		glog.V(2).Infof("Creating container %+v in pod %v", container, format.Pod(pod))
```

Comment:
Debugging only, will remove in next round.

On Fri, Apr 8, 2016 at 4:41 PM, Prashanth B <notifications@github.com>
wrote:

> In pkg/kubelet/dockertools/manager.go
> <https://github.com/kubernetes/kubernetes/pull/23567#discussion_r59086284>
> :
>
> > @@ -1877,7 +2075,7 @@ func (dm *DockerManager) SyncPod(pod *api.Pod, _ api.PodStatus, podStatus *kubec
> >  				continue
> >  			}
> >  		}
> > -		glog.V(4).Infof("Creating container %+v in pod %v", container, format.Pod(pod))
> > +		glog.V(2).Infof("Creating container %+v in pod %v", container, format.Pod(pod))
>
> Probably don't want to dump out the entire container with %+v at V(2)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/23567/files/21f0e955e3f1d05950fed76280c28d3e81d90742..ad7a96fe3b9dda804fee77e41f7e14ad0b4513d7#r59086284>
>